addons/packages/vsphere-cpi/1.22.3/bundle/config/upstream/vsphere-cpi/05-daemonset.yaml

```diff
@@ -11,6 +11,7 @@ spec:
       k8s-app: vsphere-cloud-controller-manager
   template:
     metadata:
+      annotations: {}
```

Comment:
Is this a vendored file and if so should we be modifying directly?

Reply:
I think we will need to add this otherwise there will be errors overlaying:
```
$ ytt -f addons/packages/vsphere-cpi/1.22.3/bundle/config/
ytt: Error: Overlaying (in following order: overlays/update-config.yaml, overlays/update-daemonset.yaml, overlays/update-secret.yaml):
  Document on line overlays/update-daemonset.yaml:8:
    Map item (key 'spec') on line overlays/update-daemonset.yaml:15:
      Map item (key 'template') on line overlays/update-daemonset.yaml:16:
        Map item (key 'metadata') on line overlays/update-daemonset.yaml:17:
          Map item (key 'annotations') on line overlays/update-daemonset.yaml:18:
            Expected number of matched nodes to be 1, but was 0
```

Reply:
> Could you add testing done to the PR description? [This](https://github.com/vmware-tanzu/community-edition/pull/2476#issue-1047993349) is an example for recent change in CSI.

Added the ytt result in the PR description

Reply:
If you move `#@overlay/match missing_ok=True` up one level then that error should go away. So  
```
      #@overlay/match missing_ok=True
      annotations:
        vsphere-cpi/data-values-hash: ...
```

Reply:
I see, updated, thanks!